src/types/context.ts

```diff
@@ -3,8 +3,14 @@ import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as Webhook
 import { createAdapters } from "../adapters";
 import { Env } from "./env";
 import { PluginSettings } from "./plugin-inputs";
+import { Logs } from "@ubiquity-dao/ubiquibot-logger";
 
-export type SupportedEventsU = "issue_comment.created"; // Add more events here
+/**
+ * Update `manifest.json` with any events you want to support like so:
```

Comment:
I wonder if we could directly pick up the values for the manifest to avoid mistakes? If we import the `json` the types should be working properly, what do you think?

Reply:
because the event strings are part of a string array they are typed as `string` and getting a string literal type involves fancy footwork. If they were object keys we'd have literal types for each defined without any fancy footwork.

Do you have a method for extracting json string values and converting them to literals?

Reply:
I didn't realize that arrays were not type, I expected it to export as const strings. The only options we would is to generate the code with a script, or change it to an object. But I do not see what kind of object we would have as we just require a list of strings. Let's keep it that way for now.